### DISCUSSION TITLE
Transaction Failed: Order below deleted.

### DISCUSSION THREAD

**Discussion by yocovn** [beeb/pancaketrade#42] [Q&A] [answered]
Transaction failed: 0x2baac7... (https://bscscan.com/tx/0x2baac7a3aea65fb014837f2d41669b8984d98fbb97845143b1cc752b400bcc3c)
Order below deleted.
MTV - (#1) limit sell 💰
Amount: 2,600.0 MTV
Price: > 0.0000462 BNB
Slippage: 0.10%
Gas: network default
Created: 2021-11-01 15:11

**Comment by yocovn** [comment, marked as answer]
So, it's obviously pancake router failure. Although the order should not be deleted I think...

**Comment by beeb** [comment]
Hello yocovn,

This particular error is due to insufficient slippage or big price action while the transaction was pending. https://docs.pancakeswap.finance/help/troubleshooting

If I didn't delete the order, it would potentially enter an infinite loop where it would try to submit it again and again with wrong parameters, thus wasting gas. Deleting the order is a safety measure.

**Comment by osatien** [comment]
perhaps disable it instead of deleting? it needs management but would be possible

**Comment by beeb** [comment]
Interesting idea! Can you make a feature request in the issues tracker? Something like "Add enabled/disabled status in orders instead of deleting on failure". Thanks!
